### PR TITLE
Pass the binary path of Chrome to wpt run in CI

### LINKED ISSUE #9979: wptrunner_infrastructure test job getting no results from Chrome but passing
See e.g. https://travis-ci.org/w3c/web-platform-tests/jobs/352408702

From end:

```
 0:28.80 pid:8263 Full command: /home/travis/build/w3c/web-platform-tests/_venv/bin/chromedriver --port=4445 --url-base=/
pid:8263 Starting ChromeDriver 2.36.540471 (9c759b81a907e70363c6312294d30b6ccccc2752) on port 4445
 0:28.80 pid:8263 Only local connections are allowed.
 0:29.30 INFO Starting runner
 0:29.53 WARNING Connecting to Selenium failed:
Traceback (most recent call last):
  File "/home/travis/build/w3c/web-platform-tests/tools/wptrunner/wptrunner/executors/executorselenium.py", line 53, in setup
    desired_capabilities=self.capabilities)
  File "/home/travis/build/w3c/web-platform-tests/_venv/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 154, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "/home/travis/build/w3c/web-platform-tests/_venv/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 243, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "/home/travis/build/w3c/web-platform-tests/_venv/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 312, in execute
    self.error_handler.check_response(response)
  File "/home/travis/build/w3c/web-platform-tests/_venv/lib/python2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
SessionNotCreatedException: Message: session not created exception: Chrome version must be >= 63.0.3239.0
  (Driver info: chromedriver=2.36.540471 (9c759b81a907e70363c6312294d30b6ccccc2752),platform=Linux 4.4.0-101-generic x86_64)
 0:29.54 WARNING Failed to connect to Selenium
 0:29.56 WARNING u'runner_teardown': ()
 0:29.56 ERROR Max restarts exceeded
 0:29.56 INFO Got 0 unexpected results
 0:29.56 SUITE_END

web-platform-test
~~~~~~~~~~~~~~~~~
Ran 0 checks ()
Expected results: 0
OK
```

We install the `google-chrome-unstable` package at version 66.0.3359.22-1, which I believe to be >= 63.0.3239.0.

It's also sad that this test job passes when we get zero results.

### PR DESCRIPTION
**Pull request by Hexcles**
ChromeDriver doesn't honor PATH and uses /opt/google/chrome/chrome by
default (which is usually the stable channel), so we need to explicitly
pass the desired binary path to wpt run (which in turn passes to
chromedriver via capabilities) in Travis CI (both stability check and
infrastructure test). This fixes #9979 and is one more step closer to #9932.

This PR also fixes a bug introduced in #9391. Due to the issue above, Chrome was disabled in infra integration tests, so the bug wasn't caught back then.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/10064)

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build ERRORED

Started: 2018-03-16 00:45:26
Finished: 2018-03-16 01:11:03


<h2>Failing Jobs</h2>
<ul>

<li>wpt_integration in py27,py27-flake8</li>

<li>wptrunner_infrastructure</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/26059)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/354113504)

File: tools/ci/ci_wptrunner_infrastructure.sh [hunk 1]
```diff
@@ -11,22 +11,25 @@ test_infrastructure() {
...
+    PRODUCTS=( "firefox" "chrome" )
     for PRODUCT in "${PRODUCTS[@]}"; do
         if [ "$PRODUCT" != "firefox" ]; then
             # Firefox is expected to work using pref settings for DNS
             # Don't adjust the hostnames in that case to ensure this keeps working
             hosts_fixup
         fi
         if [ "$PRODUCT" == "chrome" ]; then
-            install_chrome dev
+            test_infrastructure $(install_chrome dev)
```
Comment (gsnedders) [inline tools/ci/ci_wptrunner_infrastructure.sh]: This seems to be getting called as:

```
test_infrastructure /usr/bin/google-chrome-stable Selecting previously unselected package google-chrome-unstable. '(Reading' database ... 93729 files and directories currently 'installed.)' Preparing to unpack google-chrome-unstable_current_amd64.deb ... Unpacking google-chrome-unstable '(66.0.3359.26-1)' ... Setting up google-chrome-unstable '(66.0.3359.26-1)' ... update-alternatives: using /usr/bin/google-chrome-unstable to provide /usr/bin/google-chrome '(google-chrome)' in auto mode Processing triggers for mime-support '(3.54ubuntu1.1)' ... Processing triggers for man-db '(2.6.7.1-1ubuntu1)' ... /usr/bin/google-chrome-unstable
```

This obviously doesn't work. :)

**Review by gsnedders — CHANGES_REQUESTED** [review]

**Comment by Hexcles** [comment]
Looks like I fixed the binary path issue.

cc @kereliuk to take a look at the WebDriver error.

File: tools/wptrunner/wptrunner/executors/executorselenium.py [hunk 2]
```diff
@@ -88,7 +88,7 @@ def close_old_windows(self):
         for handle in handles:
             try:
                 self.webdriver.switch_to_window(handle)
-                self.webdriver.close_window()
+                self.webdriver.close()
```
Comment (gsnedders) [inline tools/wptrunner/wptrunner/executors/executorselenium.py]: This fixes a regression from 517c2ff02d79ae13dcd04477f3afee746d7db28e (#9391).

**Review by gsnedders — APPROVED** [review]
